src/components/MainContent.css

```diff
@@ -0,0 +1,20 @@
+.input:not(:placeholder-shown) + .label {
+  color: palevioletred;
+  top: -11px;
+  font-size: 10px;
+}
+
+.input:focus + .label {
+  color: palevioletred;
+  top: -11px;
+  font-size: 10px;
+  z-index: 50;
+  width: fit-content;
+  background-color: rgba(200, 200, 200, 0.5);
+}
+/* .label::before:focus {
```

Comment:
not meant to be here? let's delete comments unless they are truly meant to be there as comments.

Reply:
thanks, deleted!